plugin.go

```diff
@@ -142,12 +126,17 @@ func (p *Plugin) installAddonViaHelm(release *types.Release) error {
 	cb.Add(commandbuilder.Arg{Type: commandbuilder.ArgTypeRaw, Value: release.ChartPath})
 	cb.Add(commandbuilder.Arg{Type: commandbuilder.ArgTypeLongParam, Name: "version", Value: release.Version})
 
+	if p.ClusterConfig.Helm.Debug {
+		cb.Add(commandbuilder.Arg{Type: commandbuilder.ArgTypeRaw, Value: "--debug"})
+	}
+
 	// Add namespaces to command
 	if release.Namespace != "" {
 		cb.Add(commandbuilder.Arg{Type: commandbuilder.ArgTypeLongParam, Name: "namespace", Value: release.Namespace})
 	}
-	if p.ClusterConfig.Helm.Namespace != "" {
-		cb.Add(commandbuilder.Arg{Type: commandbuilder.ArgTypeLongParam, Name: "tiller-namespace", Value: p.ClusterConfig.Helm.Namespace})
+
+	if p.ClusterConfig.Helm.LogLevel != "" {
+		cb.Add(commandbuilder.Arg{Type: commandbuilder.ArgTypeLongParam, Name: "v", Value: p.ClusterConfig.Helm.LogLevel})
```

Comment:
```suggestion
		cb.Add(commandbuilder.Arg{Type: commandbuilder.ArgTypeShortParam, Name: "v", Value: p.ClusterConfig.Helm.LogLevel})
```

Reply:
Actually, you can ignore this. The `helm -h` output shows that `--v` is acceptable.

Reply:
both are supported:
```
-v, --v Level                          number for the log level verbosity
```